composer.json

```diff
@@ -18,6 +18,7 @@
     ],
     "require": {
         "php": "^8.0",
+        "illuminate/framework": "^v9.32.0",
```

Comment:
This might not be required, older versions of Laravel will not call `getClassNameForSerialization` and the functionality will remain unchanged.